### PR TITLE
Autodiscover of use cases

### PR DESCRIPTION
**Pull request by santitigaga**
Before submitting a pull request, please read
https://github.com/bancolombia/scaffold-clean-architecture/wiki/Contributing

## Category
- [x] Feature
- [ ] Fix

## Checklist
- [x] The pull request is complete according to the [guide of contributing](https://github.com/bancolombia/scaffold-clean-architecture/wiki/Contributing)
- [x] Automated tests are written
- [x] The documentation is up-to-date
- [x] the version of the readme.md, Constants.java and gradle.properties was increased
- [ ] The pull request has a descriptive title that describes what has changed, and provides enough context for the changelog

### CODE REVIEW

File: src/main/resources/structure/applications/appservice/usecases.java.mustache [hunk 1]
```diff
@@ -0,0 +1,14 @@
+package {{package}};
```
Comment (juancgalvis) [inline src/main/resources/structure/applications/appservice/usecases.java.mustache]: missing .config package at end

**Comment by juancgalvis** [comment]
Please update plugin version in the Readme file.

File: src/main/resources/structure/applications/appservice/usecases.java.mustache (hunk 1 above)
Comment (santitigaga) [reply src/main/resources/structure/applications/appservice/usecases.java.mustache]: solved

**Review by juancgalvis — APPROVED** [review]
It's ready!!

**Pull request merged**